src/libraries/System.Diagnostics.StackTrace/tests/StackFrameTests.cs

```diff
@@ -116,8 +116,8 @@ public static IEnumerable<object[]> ToString_TestData()
             yield return new object[] { new StackFrame(), "MoveNext at offset {offset} in file:line:column {fileName}:{lineNumber}:{column}" + Environment.NewLine };
             yield return new object[] { new StackFrame("FileName", 1, 2), "MoveNext at offset {offset} in file:line:column FileName:1:2" + Environment.NewLine };
             yield return new object[] { new StackFrame(int.MaxValue), "<null>" + Environment.NewLine };
-            yield return new object[] { GenericMethod<string>(), "GenericMethod<T> at offset {offset} in file:line:column {fileName}:{lineNumber}:{column}" + Environment.NewLine };
-            yield return new object[] { GenericMethod<string, int>(), "GenericMethod<T,U> at offset {offset} in file:line:column {fileName}:{lineNumber}:{column}" + Environment.NewLine };
+            yield return new object[] { GenericMethod<string>(), "GenericMethod<__Canon> at offset {offset} in file:line:column {fileName}:{lineNumber}:{column}" + Environment.NewLine };
```

Comment:
These tests are likely going to fail with native AOT or on Mono.

Reply:
There tests already failed with naot and mono even before this change. See https://github.com/mono/mono/issues/15184